firmware/Inc/AL.h

```diff
@@ -0,0 +1,75 @@
+/**
+ * AL.h
+ * Declares the Algorithm Layer of the Micromouse firmware
+ */
+
+#ifndef AL_H
+#ifdef __cplusplus
+#define AL_H
+
+#include "PHIL.h"
+
+namespace MMAL
+{
+	class Robot
```

Comment:
I think the right way to format this is to separate all of our classes into their own files. Namespace them into al, pal, or phil as needed. Is there a need for all the classes to be in the same file?